src/server/controllers/user.js

```diff
@@ -33,15 +33,17 @@ const postRegisterDetails = async (req, res) => {
 }
 
 const postLoginDetails = async (req, res) => {
+    console.log(req.body)
     try {
         const { username, password } = req.body;
         const findUser = await prisma.user.findUnique({
             where: {
                 username
             }
         });
+        console.log(findUser)
 
-        const matchingPassword = await bcrypt.compare(password, findUser.password);
+        const matchingPassword = findUser ? await bcrypt.compare(password, findUser.password) : null;
```

Comment:
You like this format haha

Reply:
lol i sure do. We need this to return either truthy or falsy; this seems to be the best way to do it as using an IF statement means declaring `matchingPassword` outside of the statement first

Reply:
Nathan and his ternaries :)